claim/validations.py

```diff
@@ -112,32 +112,31 @@ def validate_claim(claim, check_max):
 def validate_claimitems(claim):
     errors = []
     target_date = claim.date_from if claim.date_from else claim.date_to
-    for claimitem in claim.items \
-            .filter(validity_to__isnull=True) \
-            .filter(Q(rejection_reason=0) | Q(rejection_reason__isnull=True)):
-        errors += validate_claimitem_validity(claim, claimitem)
-        if not claimitem.rejection_reason:
-            errors += validate_claimitem_in_price_list(claim, claimitem)
-        if not claimitem.rejection_reason:
-            errors += validate_claimdetail_care_type(claim, claimitem)
-        if not claimitem.rejection_reason:
-            errors += validate_claimdetail_limitation_fail(claim, claimitem)
+    for claimitem in claim.items.all():
```

Comment:
Wouldn't this is result in more claim being fetched, not less?

Reply:
no it will be limited to the prefetch